spec/index.html

```diff
@@ -538,72 +538,70 @@ <h2>RDF Graphs</h2>
-      the <a>predicate</a> of an <a>RDF triple</a> may only be an <a>IRI</a>;
-      the <a>object</a> of an <a>RDF triple</a> may be either an <a>IRI</a>,
-      a <a>blank node</a>, a <a>literal</a> or a <a>quoted triple</a>.</p>
-
-    <p class="note">The definition of <a>quoted triple</a> is recursive.
-      That is, a <a>quoted triple</a> can itself have a
-      <a>subject</a> or <a>object</a> component which is another <a>quoted triple</a>.
-      However, by this definition, cycles of <a>quoted triples</a> cannot be created.</p>
+      |o| is called the <dfn>object</dfn> of the triple.
+      Similarly, given a <a>triple term</a> (|s|, |p|, |o|),
+      |s| is called the <em>subject</em> of the triple term,
+      |p| is called the <em>predicate</em> of the triple term, and
+      |o| is called the <em>object</em> of the triple term.</p>
+
+    <p class="note">While, syntactically, the notion of an <a>RDF triple</a>
+      and the notion of a <a>triple term</a> are the same, they represent
+      different concepts. RDF triples are the members of <a>RDF graphs</a>,
+      whereas triple terms can be used as components of RDF triples.</p>
+
+    <p class="note">The definition of <a>triple term</a> is recursive.
+      That is, a <a>triple term</a> can itself have an
+      <a>object</a> component which is another <a>triple term</a>.
+      However, by this definition, cycles of <a>triple terms</a> cannot be created.</p>
 
     <p><a>IRIs</a>, <a>literals</a>,
-      <a>blank nodes</a>, and <a>quoted triples</a> are collectively known as
+      <a>blank nodes</a>, and <a>triple terms</a> are collectively known as
       <span id="dfn-rdf-terms"><!-- obsolete term--></span><dfn data-lt="rdf term">RDF terms</dfn>.</p>
 
     <p><a>IRIs</a>, <a>literals</a>,
-      <a>blank nodes</a>, and <a>quoted triples</a> are distinct and distinguishable.
+      <a>blank nodes</a>, and <a>triple terms</a> are distinct and distinguishable.
       For example, a literal with the string <code>http://example.org/</code> as
       its <a>lexical form</a>
       is not equal to the IRI <code>http://example.org/</code>,
       nor to a blank node with the <a>blank node identifier</a>
       <code>http://example.org/</code>.</p>
 
     <p>The set of <span id="dfn-nodes"><!-- obsolete term--></span><dfn data-lt="node">nodes</dfn> of an <a>RDF graph</a>
-      is the set of <a>subjects</a> and <a>objects</a> of <a>triples</a> in the graph.
+      is the set of <a>subjects</a> and <a>objects</a> of the <a>triples</a> in the graph.
       It is possible for a predicate IRI to also occur as a node in
       the same graph.</p>
 
-    <p>An <dfn data-local-lt="asserted">asserted triple</dfn>
-      is an <a>RDF triple</a> that is an element of an <a>RDF graph</a>.</p>
-
-    <p class="note">In an <a>RDF graph</a>,
-      a <a>triple</a> may occur as either a <a>quoted triple</a>, an 
-      <a>asserted triple</a>, or both.</p>
```

Comment:
Why loose the notion of "asserted triple", is that not just as important with the change to triple terms?

Reply:
@gkellogg 
I don't see what introducing this notion of "asserted triple" adds; in particular now that we want to establish a clear conceptual separation between the notion of an RDF triple (or, short, triple) and the notion of a triple term. By the way the notion of "asserted triple" was defined here, whenever you want to talk about an asserted triple in a particular graph, you may as well talk about a triple in that graph.

Of course, we may include a statement somewhere, saying that every triple that is an element of an RDF graph is considered as asserted in that graph, but I don't think we need to introduce the notion of "asserted triple" to do so.